tests/cases/standalone/common/copy/copy_database_from_fs_parquet.result

```diff
@@ -6,11 +6,11 @@ INSERT INTO demo(host, cpu, memory, ts) values ('host1', 66.6, 1024, 16552765570
 
 Affected Rows: 2
 
-COPY DATABASE public TO '/tmp/demo/export/parquet/' WITH (FORMAT="parquet");
+COPY DATABASE public TO '${SQLNESS_HOME}/demo/export/parquet/' WITH (FORMAT="parquet");
```

Comment:
Why not use `data_home`? It's what it is.

Reply:
> Why not use `data_home`? It's what it is.

`data_home` is private to each datanode instance, while variable subsitution is carried out by sqlness runner, which is only aware of `sqlness_home`.